runtime/dancebox/src/lib.rs

```diff
@@ -482,13 +482,7 @@ pub struct CollatorsFromInvulnerablesAndThenFromStaking;
 
 /// Play the role of the session manager.
 impl SessionManager<AccountId> for CollatorsFromInvulnerablesAndThenFromStaking {
-    fn new_session(index: SessionIndex) -> Option<Vec<AccountId>> {
-        log::info!(
```

Comment:
Is there a reason why we are removing this log?

Reply:
Is this log useful? It get spammed in the terminal when running benchmarks.

Reply:
Hm is useful when running the regular mode I would say, maybe you can tell it not to run with runtime-benhcmarks features-?

Reply:
Yeah it is useful to me, never noticed it's spamming on benchmarks

Reply:
I actually did :)